### ISSUE TITLE
Enforce snake_case filenames.

### ISSUE THREAD

**Issue by lmwnshn** [cmu-db/terrier#771]
We don't want camelsInFilenames.

Will probably add a rule to make check-censored later.

**Comment by arshbeerSingh** [comment]
Can I work on this issue?

**Comment by lmwnshn** [comment]
@arshbeerSingh Go for it.

**Comment by arshbeerSingh** [comment]
@lmwnshn thank you very much for allowing me to work on this issue.
I was going through the files (in src) and noticed that all the file names are already in snake_case.
Can you please clarify what exactly needs to be done (if I am mistaken)?

**Comment by lmwnshn** [comment]
Yes, everything is snake_case right now. But sometimes new people don't know what the convention is, and add a new file that is camelCaseOrSomething.

We would like to prevent people from doing this as part of our continuous integration pipeline. You can see an example with `check-censored` [here](https://github.com/cmu-db/noisepage/blob/8cf1a2a664c09aba29fefe151e31341c9553cfe4/CMakeLists.txt#L1166), which uses `grep` to check the contents of files for bad words.

In this case, you want a shell script that checks all the filenames in a directory. It would probably be some combination of `find` and `grep`.
